wordpress-mobile/WordPress-iOS
# Help BuddyBuild to find the Aztec module

BuddyBuild has been failing to build our project for several days. This fixes it by explicitly adding Aztec to the `WordPressShareExtension` target.

To test:
 - Wait to see if BuddyBuild builds this branch correctly.

CC: @diego @bummytime @sergio
